docs/environment-variables.md

```diff
@@ -103,6 +103,8 @@ spec:
         - env:
           - name: WORKFLOW_GC_PERIOD
             value: 30s
+          - name: RETENTION_GC_PERIOD
+            value: 30s
```

Comment:
Minor: I don't think this is necessary. This examples shows were to add env vars in general, not GC envs specifically.

Reply:
Ok, will remove.